examples/js/modifiers/SimplifyModifier.js

```diff
@@ -94,25 +94,58 @@
 
 			}
 
+			// sort & reindex now.
+			// maintaining order by edge collapse cost will be cheaper
+			// than searching for lowest every time.
+			// Lowest cost at tail will minimize reindexing on removal.
+			vertices.sort((a,b)=>{return b.collapseCost-a.collapseCost;});
```

Comment:
You can use this tool to format the code:
https://zz85.github.io/mrdoobapproves/